### ISSUE TITLE
Cannot create migrations when using index with `where` `PQ` object.

### ISSUE THREAD

**Issue by jproffitt** [mattiaslinnap/django-partial-index#12]
I am trying to create an index like so:

```
PartialIndex(fields=['date_created', 'ignored', 'id'], unique=True, where=PQ(ignored=False))
```

But when I run `makemigrations`, django cannot serialize the `PQ` object into a migration. Here is the output:

```
ValueError: Cannot serialize: <PQ: (AND: ('ignored', False))>
There are some values Django cannot serialize into migration files.
For more, see https://docs.djangoproject.com/en/1.11/topics/migrations/#migration-serializing
```

**Comment by mattiaslinnap** [comment]
Hi @jproffitt , thanks for the report, will have a look!

Which version of Django are you using?

**Comment by jproffitt** [comment]
@mattiaslinnap thanks! I'm using django 1.11.14.

**Comment by mattiaslinnap** [comment]
@jproffitt I added tests for making and applying migrations, and fixed compatibility for Django 1.11. Please try the latest 0.5.2 release!

**Comment by jproffitt** [comment]
@mattiaslinnap works great! Thanks so much!
